code/app/src/main/java/com/example/goldencarrot/data/db/WaitListRepository.java

```diff
@@ -236,29 +233,6 @@ public void isUserInWaitList(String docId, User user, FirestoreCallback callback
                 });
     }
 
-    /**
-     * Checks the status of a user in the waitlist.
-     *
-     * @param docId the document ID of the waitlist
-     * @param user  the user to check
-     * @param callback a callback that handles the result
-     */
-    @Override
-    public void getUserStatus(String docId, UserImpl user, FirestoreCallback callback) {
```

Comment:
We don't need this anymore?

Reply:
This method is not used.

Reply:
In case we need to know a user status in the future we can always fetch the waitlist object and then check the userMap by calling `waitlist.getUserMap();`